templates/observability/index.html

```diff
@@ -0,0 +1,647 @@
+    <hr class="p-separator" />
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row u-equal-height">
+    <div class="col-4">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/3b176ef1-managed-private-cloud.svg",
+        alt="",
+        width="50",
+        height="50",
+        hi_def=True,
+        loading="lazy",
+        ) | safe
+      }}
+      <h3>99.99% uptime</h3>
+      <p>Monitoring tools with strong SLAs to build on</p>
+    </div>
+    <div class="col-4">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/81e3bf86-We+operate+your+Kubernetes.svg",
+          alt="",
+          width="50",
+          height="50",
+          hi_def=True,
+          loading="lazy",
+          ) | safe
+        }}
+      <h3>24/7 monitoring</h3>
+      <p>Active management and break/fix response by Canonical engineers</p>
+    </div>
+    <div class="col-4">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/0714f018-VMware-migration.svg",
+        alt="",
+        width="50",
+        height="50",
+        hi_def=True,
+        loading="lazy",
+        ) | safe
+      }}
+      <h3>High availability</h3>
+      <p>Fault tolerant and ready for real-world challenges</p>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-4">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/a43d61ae-update+control.svg",
+        alt="",
+        width="50",
+        height="50",
+        hi_def=True,
+        loading="lazy",
+        ) | safe
+      }}
+      <h3>Patching and updates</h3>
+      <p>All updates and security patches included</p>
+    </div>
+    <div class="col-4">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/46439b10-2.+model_AW.svg",
+          alt="",
+          width="76",
+          height="50",
+          hi_def=True,
+          loading="lazy",
+          ) | safe
+        }}
+      <h3>Transparency</h3>
+      <p>Dashboard with key metrics for each deployment</p>
+    </div>
+    <div class="col-4">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/69aaa876-compliance.svg",
+          alt="",
+          width="50",
+          height="50",
+          hi_def=True,
+          loading="lazy",
+          ) | safe
+        }}
+      <h3>Security and certifications</h3>
+      <p>ISO 27000, GDPR complaint, SOC2 Type 2</p>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--bottom u-no-padding--top">
+  <div class="u-fixed-width">
+    <h2 class="p-heading--four">We run your monitoring tools anywhere</h2>
+    <p>Your monitoring tools and monitoring dashboards, run by Canonical, wherever your applications are.</p>
+    <p><a href="/managed/contact-us" class="p-button--positive">Request a free deployment assessment</a></p>
+    <hr class="p-separator" />
```

Comment:
This separator feels too large, the "We run your monitoring tools anywhere" is substantiated by the logos of all the places we do that, and now it feels too far away in the page